### ISSUE TITLE
Known issue: UXL Launcher builds from AppVeyor may have text in the About window crammed together.

### ISSUE THREAD

**Issue by DrewNaylor** [DrewNaylor/UXL-Launcher#121]
Screenshot:
![appveyor-builds-have-messed-up-text-in-about-window](https://user-images.githubusercontent.com/15517419/41804745-ba612e32-7669-11e8-84ec-be8f574e2320.PNG)

I could have the About window replace blank lines with `vbCrLf` when getting the About window string for the textbox from My.Resources, ~~but if a user just builds it on their computer, it should be ok~~. Actually, I'm getting all the text jumbled together in a VM as well. I don't know what causes this issue, but it may be due to line ending problems.

**Comment by DrewNaylor** [comment]
This also seems to affect the theme list. I'll change the theme list to just a string instead of a file resource.

**Comment by DrewNaylor** [comment]
I may have found the problem.
![improper line endings](https://user-images.githubusercontent.com/15517419/42295144-5c3efe38-7fb5-11e8-9166-106dbc8c594f.PNG)

**Comment by DrewNaylor** [comment]
The solution would just be to replace `vbLf` with `vbCrLf` when loading the text file resources.

**Comment by DrewNaylor** [comment]
The solution would just be to replace `vbLf` with `vbCrLf` when loading the text file resources.

In the screenshot below, outlines in red are incorrect and outlines in purple are correct with the Unix line endings replaced.
![hidesettingspages replace vblf with vbcrlf](https://user-images.githubusercontent.com/15517419/42355959-cd84d9da-809d-11e8-9b74-9cc119762a2e.PNG)

**Comment by DrewNaylor** [comment]
I think this is a solution that would work. Copying from the relevant HideSettingsPages commits (https://github.com/DrewNaylor/HideSettingsPages/commit/eae7bdd22b20ebb46c1c8ea4086d36615703cade and https://github.com/DrewNaylor/HideSettingsPages/commit/d51c27c2bd1cf9409bb949334b80202b8203a768):

```
 Replace Unix line endings with Windows...
line endings when displaying the text file resource for
HideSettingsPages info. This may be related to
DrewNaylor/UXL-Launcher#121
```

```
 Replace Unix line endings with Windows...
line endings for the hsp_registry-helper info text file resource. This
makes it so that the text isn't all a mess in the About window if the
user builds from the master branch (or any other branch) hosted on
GitHub. It should display correctly with Visual Studio without doing
this, but it's better for the text to display correctly if someone wants
to test the latest code or build it from source when releasing the app.
```

**Comment by DrewNaylor** [comment]
I can fix this for version 3.1 before that's released officially. It shouldn't be that major of a change since it was easy to fix this problem in HideSettingsPages.

**Comment by DrewNaylor** [comment]
This would have actually been a regression since the About window text displays properly in version 3.0 when manually building from source.

![correct about window text line endings](https://user-images.githubusercontent.com/15517419/42356551-0effa824-80a1-11e8-8a84-7ac69912fdb5.PNG)
